packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts

```diff
@@ -337,136 +353,138 @@ describe('ThemeStore', () => {
-	// 		name: config.name,
-	// 	});
-
-	// 	// extra assertions on 'test' theme
-	// 	expect(merged.components?.results?.columns).toBe(7);
-	// });
+	it('can get theme at breakpoint and use breakpoint overrides', () => {
+		const config: ThemeStoreThemeConfig = {
+			name: GLOBAL_THEME_NAME,
+			type: 'local',
+			base: testTheme,
+			overrides: {
+				responsive: {
+					mobile: {
+						results: { columns: 7 },
+					},
+					tablet: {},
+					desktop: {},
+				},
+			},
+			variables: {
+				breakpoints: { mobile: 100, tablet: 200, desktop: 300 },
+			},
+			currency: {},
+			language: {},
+			languageOverrides: {},
+			innerWidth: 50,
+		};
 
-	//todo test
-	// it('can get theme with all the things ', () => {
-	// 	const bpIndex = 0; // simulate being at first breakpoint
-
-	// 	const config: ThemeStoreThemeConfig = {
-	// 		name: GLOBAL_THEME_NAME,
-	// 		type: 'local',
-	// 		base: testTheme,
-	// 		overrides: {
-	// 			responsive: {
-	// 				mobile: {
-	// 					results: { columns: 7 },
-	// 				},
-	// 				tablet: {},
-	// 				desktop: {},
-	// 			},
-	// 		},
-	// 		variables: {
-	// 			breakpoints: {mobile: 100, tablet: 200, desktop: 300},
-	// 		},
-	// 		currency: {
-	// 			components: {
-	// 				results: { columns: 9 },
-	// 			},
-	// 		},
-	// 		language: {
-	// 			components: {
-	// 				results: { columns: 11 },
-	// 			},
-	// 		},
-	// 		languageOverrides: {},
-	// 		innerWidth: 50,
-	// 	};
-
-	// 	const store = new ThemeStore({ config, dependencies, settings });
-	// 	expect(store.innerWidth).toBe(config.innerWidth);
-
-	// 	const baseResponsiveOverrides = config.base.responsive?.[bpIndex]!;
-	// 	expect(baseResponsiveOverrides).toBeDefined();
-
-	// 	const additionalResponsiveOverrides = config.overrides?.responsive?.[bpIndex]!;
-	// 	expect(baseResponsiveOverrides).toBeDefined();
-
-	// 	const overrideObj = { path: ['results', 'columns'], rootEditingKey: 'components', value: 12 };
-	// 	store.setOverride(overrideObj);
-
-	// 	// testing all the things!!!
-	// 	// mergeThemeLayers(base, baseResponsive, currency, language, overrides, overridesResponsive, variables, layout, editor)
-
-	// 	const merged = mergeThemeLayers(
-	// 		config.base,
-	// 		baseResponsiveOverrides,
-	// 		config.currency,
-	// 		config.language,
-	// 		config.overrides!,
-	// 		additionalResponsiveOverrides,
-	// 		{ variables: config.variables },
-	// 		store.stored
-	// 	);
-
-	// 	expect(store.theme).toStrictEqual({
-	// 		...merged,
-	// 		name: config.name,
-	// 	});
-
-	// 	// extra assertions on 'test' theme
-	// 	expect(merged.components?.results?.columns).toBe(12);
-	// });
+		const store = new ThemeStore({ config, dependencies, settings });
+		expect(store.innerWidth).toBe(config.innerWidth);
+
+		const baseResponsiveOverrides = config.base.responsive?.mobile!;
+		expect(baseResponsiveOverrides).toBeDefined();
+
+		const additionalResponsiveOverrides = config.overrides?.responsive?.mobile!;
+		expect(baseResponsiveOverrides).toBeDefined();
```

Comment:
Different variable names set to the same value - and two identical assertions... 🤔